101-bundle-json.md

```diff
@@ -13,7 +13,7 @@ The `bundle.json` file is a representation of bundle metadata. It MUST be repres
 
 A `bundle.json` is broken down into the following categories of information:
 
-- The schema version of the bundle, as a string with a `v` prefix. This schema is to be referenced as `v1` or `v1.0.0`
+- The schema version of the bundle, as a string with a `v` prefix. This schema is to be referenced as `v1` or `v1.1.0`
```

Comment:
I wasn't sure if all of these `v1.0.0` references should be updated to `v1.1.0` or not?

Reply:
While the schema has not changed, it is versioned with the core of the spec so I think it should match that (i.e.: 1.1.0).